src/gui/mzroll/eicwidget.cpp

```diff
@@ -907,31 +905,23 @@ void EicWidget::replot(PeakGroup* group) {
 			_focusLineRt = group->compound->expectedRt;
 	else _focusLineRt = 0;
 
-    if(_showCubicSpline) addCubicSpline();
-    if(_showBaseline)  addBaseLine();   //qDebug() << "\tshowBaseLine msec=" << timerX.elapsed();
-	if(_showTicLine or _showBicLine)   addTicLine();    //qDebug() << "\tshowTic msec=" << timerX.elapsed();
-    if(_showMergedEIC) addMergedEIC();
-    if(_focusLineRt >0) setFocusLine(_focusLineRt);  //qDebug() << "\tsetFocusLine msec=" << timerX.elapsed();
-    if(_showNotes)	 getNotes(eicParameters->_slice.mzmin,eicParameters->_slice.mzmax);	//get notes that fall withing this mzrange
-    if(_showMS2Events && eicParameters->_slice.mz>0) { addMS2Events(eicParameters->_slice.mzmin, eicParameters->_slice.mzmax); }
+    if (_showCubicSpline) addCubicSpline();
+    if (_showBaseline) addBaseLine();
+	if (_showTicLine || _showBicLine) addTicLine();
+    if (_showMergedEIC) addMergedEIC();
+    if (_focusLineRt >0) setFocusLine(_focusLineRt);
+    if (_showNotes)	getNotes(eicParameters->_slice.mzmin,eicParameters->_slice.mzmax);	//get notes that fall withing this mzrange
```

Comment:
The function calls should be done on the next line, no?  Our style guide does not mention whether the calls should be made in the next line or the same one but I prefer defining anything after the conditional statement on the next line,

e.g.
```
    ...

    if(_showCubicSpline)
        addCubicSpline();

    ...
```

Reply:
Agreed. I'll push the changes in this PR.
Also, please add this to the style guide if @saifulbkhan agrees?

Reply:
I agree. I will add it to the style guide.